core/workspace_comment_svg.js

```diff
@@ -145,9 +145,12 @@ Blockly.WorkspaceCommentSvg.prototype.dispose = function() {
 /**
  * Create and initialize the SVG representation of a workspace comment.
  * May be called more than once.
+ *
+ * @param {boolean=} deSelectText Text inside text area will be selected if false
```

Comment:
This comment needs to be updated to use `opt_noSelect`

Reply:
Ok now things looks better 😄